exporters/jaeger/src/main/proto/jaeger/api_v2/collector.proto

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax="proto3";
+
+package jaeger.api_v2;
+
+import "jaeger/api_v2/model.proto";
```

Comment:
Can we depend on an artifact or use git submodules to get this protos instead of copying them?

/cc @yurishkuro

Reply:
When I started this PR, it was still being discussed on the most appropriate way to handle that for OpenTelemetry's own proto files. Is there a decision made already about it? 

cc @pavolloffay